modules/tinymce/src/core/test/ts/browser/EditorUploadTest.ts

```diff
@@ -285,67 +276,99 @@ UnitTest.asynctest('browser.tinymce.core.EditorUploadTest', (success, failure) =
       editor.uploadImages(uploadDone),
       editor.uploadImages(uploadDone)
     ]).then(done, die);
-  });
+  }));
+
+  const sUploadConcurrentImagesFailWithRemove = (editor) => Log.step('TBA', 'uploadConcurrentImages fails, with remove', Step.async((done, die) => {
+    let uploadCount = 0, callCount = 0;
+
+    const uploadDone = (result: UploadResult[]) => {
```

Comment:
This really should check the undo stack count at least to make sure it's what we expected. It'd also be good to have 2 or more images that are removed to make sure only a single undo level is added.